src/obu.rs

```diff
@@ -392,11 +390,11 @@ fn parse_seq_hdr(c: &mut Rav1dContext, gb: &mut GetBits) -> Rav1dResult<Rav1dSeq
     if color_description_present != 0 {
         pri = gb.get_bits(8) as Rav1dColorPrimaries;
         trc = gb.get_bits(8) as Rav1dTransferCharacteristics;
-        mtrx = gb.get_bits(8) as Rav1dMatrixCoefficients;
+        mtrx = Rav1dMatrixCoefficients(gb.get_bits(8) as u8)
```

Comment:
this could be e.g. `Rav1dMatrixCoefficients::from_byte` and then the field does not need to be `pub`

Reply:
that can work, but you would loose information on a roundtrip `Ravid::from(David::from(x)) != x` any more (this is already partial of course, but I think otherwise the equality holds today).

Also, at this point, what is the advantage of an enum (versus a struct with constants), if in practice its variants do not map cleanly to the values it is supposed to represent?

Reply:
I *think* we've removed all of the `Dav1d` to `Rav1d` conversions, but I've left the `impl From`s and `impl TryFrom`s in case we need to add them back.  So I don't think having `Dav1d::from(Rav1d::from(x)) != x` is that bad, though you're right that it could be confusing.

The advantage, admittedly very small, of an `enum` is that it could be smaller.

Reply:
I'm happy to go with the enum, but one final thing I thought of is that if the `mtrx` ever ends up in a (debug) log, that might be confusing. The log will report that the `Reserved` matrix coefficients are used even though the source file might not actually be doing that (it's actually using a value that is not defined by the spec).

Reply:
After thinking about it some more, I think your `u8` newtype wrapper is a less risky approach.  If in the future we need to squeeze any more bytes or something, we could consider a different approach, and the newtype approach is still quite ergonomic.